core/server/data/migrations/versions/4.0/17-solve-orphaned-webhooks.js

```diff
@@ -0,0 +1,47 @@
+const {createIrreversibleMigration} = require('../../utils');
+const logging = require('../../../../../shared/logging');
+const ObjectID = require('bson-objectid');
+
+module.exports = createIrreversibleMigration(
+    async function up(knex) {
+        logging.info('Resolving the orphaned webhooks');
+
+        const orphanedWebhooks = await knex('webhooks')
+            .select('webhooks.id')
+            .leftJoin('integrations', 'integrations.id', 'webhooks.integration_id')
+            .where('integrations.id', null);
+
+        for (let i = 0; i < orphanedWebhooks.length; i++) {
+            // Create new integration
+
+            const webhook = orphanedWebhooks[i];
+            const now = knex.raw('CURRENT_TIMESTAMP');
+
+            const integration = {
+                id: ObjectID.generate(),
+                type: 'custom',
+                name: `Generated integration ${i + 1}`,
```

Comment:
@tpatel because the `name`, `slug` and `description` parts will be visible in the UI they should get a copy review from someone from design team. My suggestion, reuse `name` from the webhook if it exists and point our that the integration was generated automatically for specific webhook in the `description.

Reply:
I've asked for help for the copy 👍

Reply:
The copy has been validated by Matt and Zimo ✅